rosidl_generator_py/resource/_msg_support.c.em

```diff
@@ -22,10 +22,9 @@ nested_array_dict = {}
 @{have_not_included_string = False}@
 #include <rosidl_generator_c/string.h>
 #include <rosidl_generator_c/string_functions.h>
-
 @[  end if]@
 @{
-if not field.type.is_primitive_type() and field.type.is_array:
+if not field.type.is_primitive_type():
```

Comment:
Not sure what the reason was in the first place but it looks like that this is meant to be looking to nested arrays (c.f. variable name below `nested_array_dict`)